python/taichi/main.py

```diff
@@ -224,37 +224,31 @@ def example(self, arguments: list = sys.argv[2:]):
     def changelog(self, arguments: list = sys.argv[2:]):
         """Display changelog of current version"""
         parser = argparse.ArgumentParser(
-            prog='ti changelog', description=f"{self.changelog.__doc__}")
-        import taichi as ti
-        if ti.is_release():
-            args = parser.parse_args(arguments)
-            changelog_md = os.path.join(ti.package_root(), 'CHANGELOG.md')
-            with open(changelog_md) as f:
-                print(f.read())
+            prog='ti changelog',
```

Comment:
Note this is likely no longer needed, if you type `ti changelog` as a user, you just want to read the changelog installed in the package. You don't really want to write a changelog as user. 
In other words we might safely move the writing part in this function to a helper script (e.g. make_changelog.py). It's even better if we just move make_changelog.py out of `python/taichi` folder to e.g. `misc` folder (so that we don't release it to users).